assets/ocp-performance/queries.libsonnet

```diff
@@ -162,15 +162,15 @@ local generateTimeSeriesQuery(query, legend) = [
   },
   ovnAnnotationLatency: {
     query():
-      generateTimeSeriesQuery('histogram_quantile(0.99, sum by (pod, le) (rate(ovnkube_controller_pod_creation_latency_seconds_bucket[$interval]))) > 0', '{{ pod }}'),
+      generateTimeSeriesQuery('histogram_quantile(0.99, sum by (instance, pod, le) (rate(ovnkube_controller_pod_creation_latency_seconds_bucket[$interval]))) > 0', '{{ pod }} - {{ instance }}'),
```

Comment:
Added a label to the OVN Latency panels, to help trace the pods to nodes without having to query the API. Assumes we know a node's IP address.